Frontend/Pages/Projects/TransferDates/Target.cshtml.cs

```diff
@@ -40,15 +40,16 @@ public async Task<IActionResult> OnGetAsync()
         
         public async Task<IActionResult> OnPostAsync()
         {
```

Comment:
Can also use skip on customize validator if you want to validate manually instead of startup
https://docs.fluentvalidation.net/en/latest/aspnet.html#validator-customization

Reply:
Thanks! Can you check whether the latest commit looks okay?